tests/Tests/200__monitor_and_manage/200__metrics/200__metrics.robot

```diff
@@ -0,0 +1,25 @@
+*** Settings ***
+Resource    ../../../Resources/ODS.robot
+
+*** Variables ***
+
+@{RECORD_GROUPS}  Availability Metrics  SLOs - JupyterHub  SLOs - JupyterHub DB  SLOs - ODH Dashboard  SLOs - RHODS Operator  SLOs - Traefik Proxy  Usage Metrics
+
+@{ALERT_GROUPS}  Builds  DeadManSnitch  RHODS-PVC-Usage  SLOs-haproxy_backend_http_responses_total  SLOs-probe_success
+
+*** Test Cases ***
+
+Test Existence of Prometheus Alerting Rules
+  [Tags]  Smoke  ODS-509
+  Check Prometheus Alerting Rules
+
+Test Existence of Prometheus Recording Rules
+  [Tags]  Smoke  ODS-510
```

Comment:
Sorry, in the end it's better that this goes to Sanity first
```
 [Tags]  Sanity  ODS-510
```

Reply:
Fixed!